site/content/en/docs/concepts/admission_fair_sharing.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Admission Fair Sharing"
+date: 2025-05-28
+weight: 6
+description: >
+  A mechanism allowing to ordering Workloads based on the historical resource usage of their source LocalQueues, giving preference to those that have consumed less resources over time.
+---
+
+# Admission Fair Sharing
+
+Admission Fair Sharing is a mechanism introduced in Kueue v0.12.0 that helps distribute resources fairly between multiple LocalQueues targeting the same ClusterQueue. It orders workloads based on the historical resource usage of their source LocalQueues, giving preference to those that have consumed less resources over time.
+
+## How it works
+
+When multiple workloads compete for resources within a ClusterQueue:
+
+1. Kueue tracks resource usage history for each LocalQueue
+2. Workloads from LocalQueues with lower historical usage get admitted before those from high-usage queues
+3. Usage values decay over time based on configurable parameters
+
+## Configuration
+
+
+### Feature Gate
+
+To use Admission Fair Sharing, the `AdmissionFairSharing` feature gate must be enabled. You can enable it by:
```

Comment:
We should mention that this feature is alpha and disabled by default.

Reply:
Added!